### PR TITLE
MrVI Quick Start Tutorial

### PR DESCRIPTION
**Pull request by justjhong**
## Formatting

- [x] My tutorial has only one top-level (`#`) header

## Reproducibility

- [x] My tutorial works on Google Colab
- [x] My tutorial sets `scvi.settings.seed = 0` at the beginning of the notebook
- [x] My tutorial has been run and includes outputs (e.g. plots, tables)

## Other

- [ ] Counts and normalized data should co-exist in the datasets, see the [API overview](https://docs.scvi-tools.org/en/stable/tutorials/notebooks/api_overview.html) for an example
- [ ] For scRNA-seq data, normalization should be counts per median library size and then log1p transformed -- if not, a reason should be given

### CODE REVIEW

**Comment by canergen** [comment]
Does the missing Colab check mean that it doesn't work or that it's not tested yet? In the first case can we make it pass? IMO it's the easiest way for reviewers to check algorithms and for users to get to know them.

**Review by martinkim0 — CHANGES_REQUESTED** [review]
Thanks for much Justin! Overall looks great, just a couple of comments:

Could you change the first line to:

> MrVI (Multi-resolution Variational Inference) is a **model**...

and maybe put like 1-2 sentences summarizing for what kind of datasets MrVI is appropriate?

Would it also be possible to add a citation for the dataset, a quick description of how it was subsetted, as well as a summary of the different `obs` fields?

Finally, in `Visualize cell embeddings and sample distances`, could you summarize the difference between `u` and `z` (I guess I'm assuming a user who is going to this tutorial directly without having read the paper).

**Review by martinkim0 — APPROVED** [review]
LGTM!!

**Comment by martinkim0** [comment]
While you're at it, would you mind updating https://github.com/scverse/scvi-tutorials/blob/main/.github/workflows/run_linux_cuda.yml and https://github.com/scverse/scvi-tutorials/blob/main/.github/workflows/run_linux_cuda_branch.yml to include the new file? Thanks
